src/pages/class/components/TabWrapper/TabWrapper.tsx

```diff
@@ -32,7 +41,7 @@ const TabWrapper = ({ colorScheme, lessonData }: TabWrapperPropTypes) => {
       <div className={styles.tabPanelStyle}>
         {CLASS_TABS.map((tab) => (
           <TabPanel key={tab.id} isSelected={selectedTab === tab.id - 1}>
-            {tab.component(lessonData)}
+            {tab.isImplemented ? tab.component(lessonData) : null}
```

Comment:
```suggestion
{tab.isImplemented && tab.component(lessonData)}
```

해당 로직의 경우에는 `&&`를 써도 될 것 같아요!